src/doc/en/installation/conda.rst

```diff
@@ -9,48 +9,32 @@ SageMath can be installed on Linux and macOS via Conda from the
 Both the ``x86_64`` (Intel) architecture and the ``arm64``/``aarch64``
 architectures (including Apple Silicon, M1, M2, M3, M4) are supported.
 
-You will need a working Conda installation: either Miniforge (or Mambaforge),
-Miniconda or Anaconda. If you don't have one yet, we recommend installing
-`Miniforge <https://github.com/conda-forge/miniforge>`_ as
-follows. In a terminal,
+You will need a working Conda installation: either Miniforge, Miniconda or
+Anaconda. If you don't have one yet, we recommend installing `Miniforge
+<https://github.com/conda-forge/miniforge>`_ as follows. In a terminal,
 
 .. code-block:: shell
 
-   $ curl -L -O "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
-   $ bash Miniforge3-$(uname)-$(uname -m).sh
+    $ curl -L -O "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
+    $ bash Miniforge3-$(uname)-$(uname -m).sh
 
-* Miniforge (and Mambaforge) use conda-forge as the default channel.
-
-* If you are using Miniconda or Anaconda, set it up to use conda-forge:
+* Miniforge uses conda-forge as the default channel. However, if you are using
+  Miniconda or Anaconda, set it up to use conda-forge:
 
   * Add the conda-forge channel: ``conda config --add channels conda-forge``
 
   * Change channel priority to strict: ``conda config --set channel_priority strict``
 
-If you installed Miniforge (or Mambaforge), we recommend to use
-`mamba <https://mamba.readthedocs.io/en/latest/index.html>`_ in the following,
-which uses a faster dependency solver than ``conda``.
-
 .. _sec-installation-conda-binary:
 
 Installing all of SageMath from conda (not for development)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Create a new conda environment containing SageMath, either with ``mamba`` or ``conda``:
-
-.. tab:: mamba
-
-  .. code-block:: shell
+Create a new conda environment containing SageMath:
 
-      $ mamba create -n sage sage python=X
-
-.. tab:: conda
-
-  .. code-block:: shell
-
-      $ conda create -n sage sage python=X
+.. code-block:: shell
 
-where ``X`` is version of Python, e.g. ``3.9``.
+    $ conda create -n sage sage
```

Comment:
If you only keep one, I recommend using mamba. It's drop-in replacement and much faster.

Reply:
I personally use mamba but conda works just fine and it is available on the setups that we are covering here (see the top) i.e., also on miniconda and anaconda.

Reply:
I agree with maintaining conda instructions as I think it's a bit more widely available in various distro repositories.